client/my-sites/stats/stats-no-content-banner/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { translate } from 'i18n-calypso';
+import writePost from 'calypso/assets/images/onboarding/site-options.svg';
+import Banner from 'calypso/components/banner';
+import { useHasNeverPublishedPost } from 'calypso/data/stats/use-has-never-published-post';
+
+import './style.scss';
+
+interface StatsNoContentBannerProps {
+	siteSlug: string;
+	siteId: number;
+}
+
+export const StatsNoContentBanner = ( {
+	siteId,
+	siteSlug,
+}: StatsNoContentBannerProps ): JSX.Element | null => {
+	const {
+		data: hasNeverPublishedPost,
+		isLoading: isHasNeverPublishedPostLoading,
+	} = useHasNeverPublishedPost( siteId ?? null, true, {
+		retry: false,
+	} );
+
+	if ( ! hasNeverPublishedPost || isHasNeverPublishedPostLoading ) {
+		return null;
+	}
+
+	return (
+		<Banner
+			callToAction={ translate( 'Write a post' ) }
+			className="stats-no-content-banner__banner"
+			description={ translate(
+				'Sites with content get more visitors. We’ve found that adding some personality and introducing yourself is a great start to click with your audience.'
+			) }
+			disableCircle="true"
```

Comment:
We could simplify this with:

```suggestion
			disableCircle
```

Or at least pass a boolean rather than a string:

```suggestion
			disableCircle={ true }
```